docs/development.md

```diff
@@ -153,7 +153,7 @@ started prerequisites section](./getting-started.md#Prerequisites)
 
 #### Creating a dev cluster
 
-The steps below are provided in a convenient script in [hack/create-dev-cluster.sh](hack/create-dev-cluster.sh). Be sure to set `REGISTRY="<container-registry>"`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
+The steps below are provided in a convenient script in [hack/create-dev-cluster.sh](hack/create-dev-cluster.sh). Be sure to set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
```

Comment:
you can remove line 168 as well
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/509/files#diff-1bf07101845de6da98f1c948b82c5ce0R168

Reply:
or not remove but add a comment next to it saying it's optional maybe, same as the TAG variable

Reply:
I don't think we can yet, at least not with this change.

Reply:
oops yeah you're right my bad